packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5809,6 +5822,11 @@ export default class Meeting extends StatelessWebexPlugin {
     this.video = this.video || createMuteState(VIDEO, this, this.mediaProperties.mediaDirection);
     // Validation is already done in addMedia so no need to check if the lenght is greater then 0
     this.setLocalTracks(localStream);
+    if (this.isMultistream && localShare) {
+      throw new Error(
+        'calling addMedia() with localShare stream is not supported when using multistream'
```

Comment:
potentially wrong error message? shouldn't it say `preMedia`?

Reply:
no, this is intentioanal, because preMedia() is a private method and the error talks about the public API that the app must have called to reach here and cause this error - it's only possible to do that via addMedia()